tests/spring-boot/pom.xml

```diff
@@ -0,0 +1,287 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>hawtio-tests</artifactId>
+    <groupId>io.hawt.tests</groupId>
+    <version>2.18-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>hawtio-tests-springboot</artifactId>
+  <name>${project.artifactId}</name>
+  <description>hawtio :: Spring Boot 2.x tests</description>
+  <packaging>jar</packaging>
+
+  <properties>
+    <!-- Jetty version referred from the spring-boot-dependencies project -->
+    <jetty.version>9.4.11.v20180605</jetty.version>
+    <hawtio.url>/actuator/hawtio</hawtio.url>
+    <hawtio.managementPort>10001</hawtio.managementPort>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-dependencies</artifactId>
+        <version>${spring-boot-version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.camel.springboot</groupId>
+        <artifactId>camel-spring-boot-dependencies</artifactId>
```

Comment:
please use  `camel-spring-boot-bom`

Reply:
Good catch. Unrelated to this PR but we should update the other parts of Hawtio e.g. examples as well.

Reply:
Thank you, I have fixed it in tests/spring-boot/pom.xml file

Reply:
eventually camel-spring-boot-dependencies will be deprecated and removed, but sint camel-managment and camel-debug are needed and can be used with camel-spring-boot OOB, we should find a workaround or update the camel-spring-boot-bom, otherwise we need to specify the version manually https://github.com/hawtio/hawtio/pull/2788/files#diff-3fbc2050d8ed3e423c8efca95666424810c9c501e062e641be0200e02869b75dR141